wg-serving/charter.md

```diff
@@ -0,0 +1,87 @@
+# WG Serving Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+Discuss and enhance the support for AI/ML inference workloads in Kubernetes.
+
+
+### In scope
+
+- Gather requirements for serving workloads (inference primarily, but benefiting
+  other non-batch use cases where possible) that have broad community alignment
+  from practitioners, distros, and vendors. Provide concrete input to other SIGs
+  and WGs around needs for identified requirements. Do it in partnership
+  with existing ecosystem projects like kServe, Seldon, Kaito, and
+  others to identify, extract, or implement common shared problems (like Kueue
+  abstracted deferred scheduling for multiple batch frameworks).
+- Specific areas of improvement include:
+  - Directly improve key kubernetes workload controllers when used with
+    accelerators and the most common inference serving frameworks and model
+    servers.
+  - Explore new projects that improve orchestration, scaling, and load balancing
+    of inference workloads and compose well with other workloads on Kubernetes
+  - Being able to run serving workloads safely while giving up
+    available slack capacity to batch frameworks
+
+### Out of scope
+
+- Training and batch inference, which are covered by WG Batch.
+- Addition of new API kinds that serve a specific models. The focus should be on
```

Comment:
I don't know what this means, or if I agree.  I think we should be focused on CRDs that make inference work consistently on Kube and enable it.  What's the history on this one?

Reply:
I agree with Clayton, I'd drop this point entirely.

Reply:
removed